<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>437</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>613</width>
    <height>437</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>613</width>
    <height>437</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="PCM_data_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PCM数据参数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="WAV_data_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WAV输出参数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>88</y>
      <width>321</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="PCM_param_horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="input_sample_rate_verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="input_sample_rate_label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>采样率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="input_sample_rate">
         <item>
          <property name="text">
           <string>48khz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>44.1khz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16khz</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="input_sample_width_verticalLayout">
       <item>
        <widget class="QLabel" name="input_sample_width_label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>采样位深</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="input_sample_width">
         <item>
          <property name="text">
           <string>16bit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8bit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32bit</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="input_channel_verticalLayout">
       <item>
        <widget class="QLabel" name="input_channel_label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="input_sample_channel">
         <item>
          <property name="text">
           <string>2channel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1channel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4channel</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="WAV_param_horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="output_sample_rate_verticalLayout">
       <item>
        <widget class="QLabel" name="output_sample_rate_label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>采样率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="output_sample_rate">
         <item>
          <property name="text">
           <string>48khz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>44.1khz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16khz</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="output_sample_width_verticalLayout">
       <item>
        <widget class="QLabel" name="output_sample_width_label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>采样位深</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="output_sample_width">
         <item>
          <property name="text">
           <string>16bit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8bit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32bit</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="output_channel_verticalLayout">
       <item>
        <widget class="QLabel" name="output_channel_label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>通道数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="output_sample_channel">
         <item>
          <property name="text">
           <string>2channel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1channel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4channel</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="output_message">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>211</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menufile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="open_file"/>
   </widget>
   <addaction name="file_menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_file">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>input_sample_channel</sender>
   <signal>textActivated(QString)</signal>
   <receiver>output_sample_channel</receiver>
   <slot>setCurrentText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>output_sample_channel</sender>
   <signal>textActivated(QString)</signal>
   <receiver>input_sample_channel</receiver>
   <slot>setCurrentText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_sample_width</sender>
   <signal>textActivated(QString)</signal>
   <receiver>output_sample_width</receiver>
   <slot>setCurrentText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>output_sample_width</sender>
   <signal>textActivated(QString)</signal>
   <receiver>input_sample_width</receiver>
   <slot>setCurrentText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_sample_rate</sender>
   <signal>textActivated(QString)</signal>
   <receiver>output_sample_rate</receiver>
   <slot>setCurrentText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>output_sample_rate</sender>
   <signal>textActivated(QString)</signal>
   <receiver>input_sample_rate</receiver>
   <slot>setCurrentText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
